include/staff/department.inc.php

```diff
@@ -115,6 +115,19 @@
                 </span>
             </td>
         </tr>
+
+        <tr>
+            <td><?php echo __('Ticket Unassignment'); ?>:</td>
+            <td>
+                <span>
+                <input type="checkbox" name="unassign_tickets_dept" <?php echo
+                $info['unassign_tickets_dept']?'checked="checked"':''; ?>>
+                <?php echo __('Unassign ticket assignment on department transfer'); ?>
```

Comment:
I think we should make this clear that this setting is applied when a ticket is transfer *into* this department. (Rather than when a ticket is transferred *away* from the department).